backend/app/views/spree/admin/payment_methods/edit.html.erb

```diff
@@ -6,13 +6,11 @@
   <%= back_to_list_button(Spree::PaymentMethod.model_name.human, admin_payment_methods_path) %>
 <% end %>
 
-<%= render :partial => 'spree/admin/shared/error_messages', :locals => { :target => @payment_method } %>
+<%= render partial: 'spree/admin/shared/error_messages', locals: { target: @payment_method } %>
 
-<%= form_for @payment_method, :url => admin_payment_method_path(@payment_method) do |f| %>
+<%= form_for @payment_method, url: admin_payment_method_path(@payment_method) do |f| %>
   <fieldset>
-    <%= render :partial => 'form', :locals => { :f => f } %>
-    <div data-hook="buttons" class="actions">
-      <%= button Spree.t('actions.update'), 'save' %>
```

Comment:
@Mafi88 It won't since `<%= render partial: 'spree/admin/shared/edit_resource_links' %>` always adds a 'Update' button.

PS: `new_resource_links` gives 'Create' button. :p